components/post_view/post_attachment_opengraph/post_attachment_opengraph.jsx

```diff
@@ -160,25 +208,35 @@ export default class PostAttachmentOpenGraph extends React.PureComponent {
             imageUrl && renderingForLargeImage === this.state.hasLargeImage &&
             (!renderingForLargeImage || (renderingForLargeImage && this.props.isEmbedVisible))
         ) {
-            if (renderingForLargeImage) {
-                const imageDimensions = getFileDimensionsForDisplay(metadata.images && metadata.images[imageUrl], MAX_DIMENSIONS_LARGE_IMAGE);
-
-                element = (
-                    <img
-                        className={'attachment__image attachment__image--opengraph large_image'}
-                        src={imageUrl}
-                        {...imageDimensions}
-                    />
-                );
-            } else {
-                const imageDimensions = getFileDimensionsForDisplay(metadata.images && metadata.images[imageUrl], MAX_DIMENSIONS_SMALL_IMAGE);
-                element = this.wrapInSmallImageContainer(
-                    <img
-                        className={'attachment__image attachment__image--opengraph'}
-                        src={imageUrl}
-                        {...imageDimensions}
-                    />
-                );
+            if (this.props.post.metadata || (!this.props.post.metadata && this.state.imageLoaded)) {
```

Comment:
This can be simplified by removing the `!metadata`
```suggestion
            if (this.props.post.metadata || this.state.imageLoaded) {
```